frontend/lib/src/components/core/Block/styled-components.ts

```diff
@@ -67,9 +67,6 @@ export const StyledElementContainer = styled.div<StyledElementContainerProps>(
     position: "relative",
 
     "@media print": {
-      "@-moz-document url-prefix()": {
-        display: "block",
-      },
```

Comment:
This style seemed to have caused flakiness on Firefox when executing the hello_app print playwright test. It was added ~4 years ago in this change: https://github.com/streamlit/streamlit/pull/2353/files#diff-b0ce3cff40c3da251d8c22046d22ca55f73aee209f07098d1e77fc3870ea5d18R26-R28
Its unclear what this was added for and I was not able to detect any change when removing it. Also, it seems to have only affected the firefox style anyways.

Reply:
Thanks for fixing this! How did you figure this out -- did you look through the CSS to see if there was anything that could be causing the shift?

Reply:
It seems this rule using `@-moz` targets Firefox specifically. I wonder if we should be removing this one as well https://github.com/streamlit/streamlit/pull/2353/files#diff-48b9588e310cebd3348cd206d45dfbd99a3233a82fb2fa5035c2b5c927c1accfR54-R59

Does it make sense that we want to target some CSS specifically to Firefox?

Reply:
I have searched for it, but it does not seem to exist anymore. I also talked to Ken and he also thinks that this was probably a very old artifact that seems to not be needed anymore. Maybe Firefox behaved differently back then or so, but I agree with you that we should try to avoid browser-specific stylings.